### PR TITLE
Liq 1.2

### PR DESCRIPTION
**Pull request by gbalabasquer**

### CODE REVIEW

File: src/DssProxyActions.t.sol [hunk 1]
```diff
@@ -935,8 +935,9 @@ contract DssProxyActionsTest is DssDeployTestBase, ProxyCalls {
     }
 
     function _flipETH() internal returns (uint cdp) {
-        this.file(address(cat), "ETH", "lump", 1 ether); // 1 unit of collateral per batch
-        this.file(address(cat), "ETH", "chop", ONE);
+        this.file(address(cat), "ETH", "dunk", rad(200 ether)); // 200 units of DAU per batch
```
Comment (lucas-manuel) [inline src/DssProxyActions.t.sol:938]: `// 200 units of DAI per batch`

File: src/DssProxyActions.t.sol [hunk 2]
```diff
@@ -972,8 +973,9 @@ contract DssProxyActionsTest is DssDeployTestBase, ProxyCalls {
     }
 
     function testExitGemAfterFlip() public {
-        this.file(address(cat), "COL", "lump", 1 ether); // 1 unit of collateral per batch
-        this.file(address(cat), "COL", "chop", ONE);
+        this.file(address(cat), "COL", "dunk", rad(40 ether)); // 1oo units of DAI per batch
```
Comment (lucas-manuel) [inline src/DssProxyActions.t.sol:976]: `// 100 units of DAI per batch`

File: src/DssProxyActions.t.sol (hunk 1 above)
Comment (gbalabasquer) [reply src/DssProxyActions.t.sol:938]: oops

Comment (gbalabasquer) [reply src/DssProxyActions.t.sol:938]: Luckily I didn't refer to it as "DIA".

Comment (lucas-manuel) [reply src/DssProxyActions.t.sol]: Pushed changes

**Pull request merged**